pkgs/development/interpreters/python/cpython/2.7/default.nix

```diff
@@ -296,8 +298,10 @@ in with passthru; stdenv.mkDerivation ({
         # First we delete all old bytecode.
         find $out -name "*.pyc" -delete
         '' + optionalString rebuildBytecode ''
-        # Then, we build for the two optimization levels.
-        # We do not build unoptimized bytecode, because its not entirely deterministic yet.
+        # We build 3 levels of optimized bytecode. Note the default level, without optimizations,
+        # is not reproducible yet. https://bugs.python.org/issue29708
+        # Not creating bytecode will result in a large performance loss however, so we do build it.
```

Comment:
Reading https://bugs.python.org/issue29708, it sounds like it is "just" that the pyc bytecode includes the py's mtime.

How about adding a:
```
find $ount -name '*.py' -exec touch --date=@0 {} \;
```

Reply:
tried it, does not work.